packages/color-matching-block/src/settings.tsx

```diff
@@ -0,0 +1,124 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AssetChooserObjectType, FileExtension } from '@frontify/app-bridge';
+import { AssetInputSize, DropdownSize } from '@frontify/fondue';
+import { AssetInputMode, AssetInputSource } from '@frontify/guideline-blocks-settings';
+import { IconRender } from '../foundation/IconRender';
+import { IconToolbar } from '../foundation/IconToolbar';
+import { ColorMatchingType } from './types';
+
+export const ICON_LEFT_ID = 'iconLeft';
+export const ICON_RIGHT_ID = 'iconRight';
+export const LOGO_ID = 'logoId';
+export const IS_TOOLBAR_TYPE = 'toolbarType';
+
+export const settings = {
+    main: [
+        {
+            id: IS_TOOLBAR_TYPE,
+            type: 'dropdown',
+            size: 'Large' as DropdownSize.Large,
```

Comment:
Instead of casting is it possible to simply do
`size: DropdownSize.Large` ?
Same for the svg ones below